Shared/Coordinator/MainCoordinator.swift

```diff
@@ -18,7 +18,9 @@ class MainCoordinator: Coordinator<Void> {
         super.start()
 
         SessionManager.shared.didReceiveInvalidSessionError = { [unowned self] _ in
-            self.showLogOutAlert()
+            Task.onMainActor {
+                self.showLogOutAlert()
+            }
```

Comment:
This was crashing when it tried to present the alert because it wasn't on the main thread.